### ISSUE TITLE
Scality S3 Server Containers constantly restarting

### ISSUE THREAD

**Issue by nt-owen** [scality/S3#99]
I have a Scality RING with the Scality S3 Server running on 5 of the nodes.  It was running fine until I shut the whole RING down last week and now that I’ve started it up again the Docker Containers are permanently in a ‘Restarting’ state.  

[root@ip-* ~]# docker ps
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS                         PORTS               NAMES
a65145df9f5a        scality/vault-md:GA1.0.3   "/bin/sh -c 'bash -c "   2 weeks ago         Restarting (1) 6 minutes ago                       scality-vault-*
27ab662d9c47        scality/s3:GA1.0.3         "/bin/sh -c 'bash -c "   2 weeks ago         Restarting (1) 6 minutes ago                       scality-s3-*
426dc4bcf067        scality/metadata:GA1.0.3   "supervisord -c /conf"   2 weeks ago         Restarting (2) 6 minutes ago                       scality-metadata-vault-*
944ece3f67c0        scality/metadata:GA1.0.3   "/bin/sh -c 'npm star"   2 weeks ago         Restarting (1) 6 minutes ago                       scality-metadata-*
9d50016d1d4d        scality/sproxyd-tengine    "/bin/sh -c 'supervis"   2 weeks ago         Restarting (2) 6 minutes ago                       scality-sproxyd-*

If I stop Docker and then start it again the Containers all start again but still say ‘Restarting’.  The behaviour is the same on all 5 nodes.

**Comment by DavidPineauScality** [comment]
Please provide, while hiding any sensitive information:
1. docker logs output of each type of container (there may be only one or multiple reasons for this failure)
2. Tell us whether the stderr logs contain any relevant information

If none of those two methods provide any useful information, then we'll provide you with instructions for the next steps to investigate.

**Comment by nt-owen** [comment]
Thanks!  I've attached the log files of each type of container from one of the nodes.  All 5 nodes have the same content in the logs.

For each of these logs if I do:-
docker logs <container>
I get the logs but I get a number of lines saying this to stderr:-

bash: /conf/env: No such file or directory
or
Error: could not find config file /conf/supervisor/supervisord.conf

So that could be the root of the problem: that there is supposed to be something mounted at /conf and it's not.

One of the metadata Container logs also produced some errors for npm to stderr when docker logs was run against it and I've attached that as well.

There is also a container called "scality/elk" running on the Supervisor but the log file is 85MB.  I've attached the first 1000 lines and the last 1000 lines because most of it is the same content.

[docker_container_logs.zip](https://github.com/scality/S3/files/388099/docker_container_logs.zip)

Contents:-
-rw-r--r-- 1 centos centos 6.3K Jul 28 09:49 node01_metadata:GA1.0.3_426dc4bcf067_log
-rw-r--r-- 1 centos centos  42K Jul 28 09:49 node01_metadata:GA1.0.3_944ece3f67c0_log
-rw-r--r-- 1 centos centos 203K Jul 28 09:49 node01_metadata:GA1.0.3_944ece3f67c0_stderr
-rw-r--r-- 1 centos centos  16K Jul 28 09:49 node01_s3:GA1.0.3_27ab662d9c47_log
-rw-r--r-- 1 centos centos  18K Jul 28 09:49 node01_sproxyd-tengine_9d50016d1d4d_log
-rw-r--r-- 1 centos centos 9.9K Jul 28 09:49 node01_vault-md:GA1.0.3_a65145df9f5a_log
-rw-r--r-- 1 centos centos  85M Jul 28 09:40 supervisor_elk_6dd5dfb25502_log
-rw-r--r-- 1 centos centos 247K Jul 28 09:43 supervisor_elk_6dd5dfb25502_log_first_1000_lines
-rw-r--r-- 1 centos centos 155K Jul 28 09:43 supervisor_elk_6dd5dfb25502_log_last_1000_lines
